meta-resin-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -101,6 +101,20 @@ if [ ! -d "$RESIN_BOOT_MOUNTPOINT" ]; then
 	fail "$RESIN_BOOT_MOUNTPOINT : No such directory."
 fi
 
```

Comment:
Small type on commit log:
> don't have a change to complete

should be `chance` there

Reply:
Fixed